scripts/otel/otel_exporter.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jun  7 14:32:56 2024
+
+@author: massi
+"""
+
+from opentelemetry import metrics
```

Comment:
I believe it makes sense to collect the metrics rather in the orchestration script than here. The purpose if this repository is to not lay out some internal scripts and components, but simply show our clients the examples, how to basically use UH